docs/ramalama.1.md

```diff
@@ -109,6 +109,13 @@ The default can be overridden in the ramalama.conf file or via the the
 RAMALAMA_IMAGE environment variable. `export RAMALAMA_TRANSPORT=quay.io/ramalama/aiimage:latest` tells
 RamaLama to use the `quay.io/ramalama/aiimage:latest` image.
 
+#### **--keep-groups**
```

Comment:
I would prefer to have RamaLama figure this out. or always set it when Podman is used in rootless mode.
Users are not going to understand when and when not to use this.

Reply:
The downside of using it all of the time is a leak of GIDs into the container when not necessary.  Add option to allow users to disable it for security purposes,  But I think we should enable it by default for rootless containers so that users will not stumble upon it.